tests/manual/twostepscarret.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Incorrectly named – `two-step-caret.js` is the right name (because these are three separate words).